extensions/2.0/Vendor/EXT_manifold/schema/glTF.EXT_manifold.schema.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "EXT_mesh_manifold extension",
+    "type": "object",
+    "description": "glTF extension defines manifoldness for a mesh.",
+    "allOf": [
+        {
+            "$ref": "glTFProperty.schema.json"
+        }
+    ],
+    "properties": {
+        "manifoldPrimitive": {
+            "allOf": [
+                {
+                    "$ref": "mesh.primitive.schema.json"
+                },
+                {
+                    "material": false
+                },
+                {
+                    "targets": false
+                }
+            ],
+            "description": "An alternative single primitive that contains only POSITION, providing the same geometry as all of the normal primitives, but manifold."
```

Comment:
Likewise in VSCode, this `description` and any `title` added here is completely overridden by the ones defined in `mesh.primitive.schema.json`.  I don't think this should hold up the PR, but it's unfortunate if we wanted VSCode support for this.

For example here's a hover-tooltip with the incorrect description applied:

![screenshot](https://github.com/KhronosGroup/glTF/assets/1235080/59e23026-7d67-4017-aa51-b4d129461e27)